### ISSUE TITLE
Troubles to find UA with browserify/reactjs?

### ISSUE THREAD

**Issue by kud** [faisalman/ua-parser-js#84]
I have no clue why but with browserify/reactjs, I had to do this:

``` javascript
var parser = new UAParser()
parser.setUA(window.navigator.userAgent) // mandatory

var result = parser.getResult()
```

without setting ua myself via `setUA`, it returns `undefined` everywhere.

**Comment by jtwalters** [comment]
It's true, I had the same issue. Wish I had looked here first!

I believe that in browserify, `this` is the same as `exports`.

**Comment by faisalman** [comment]
Fixed in #119
